pages/agent/v3/cli_artifact.md

```diff
@@ -289,3 +290,75 @@ up, see our [Google Cloud installation guide](/docs/agent/v3/gcloud#uploading-ar
 You can configure the `buildkite-agent artifact` command to store artifacts in
 Artifactory. For instructions for how to set this up, see our
 [Artifactory guide](/docs/integrations/artifactory).
+
+## Using your private Azure Blob container
+
+>📘 Support for uploading artifacts to Azure Blob Storage was added in Agent
+> v3.53.0.
```

Comment:
The formatting of this note is a bit funny because the line break makes part of it a heading.

![Screenshot 2023-08-31 at 2 53 07 pm](https://github.com/buildkite/docs/assets/119824349/3acadcac-0b71-4b14-b4ff-d43ab270e997)

Either remove the line break so it all displays the same:

```suggestion
>📘 Support for uploading artifacts to Azure Blob Storage was added in Agent v3.53.0.
```

Or, just add this as a regular sentence in the intro:

> You can configure the `buildkite-agent artifact` command to store artifacts in your private [Azure Blob Storage container](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction). Support for uploading artifacts to Azure Blob Storage was added in Agent v3.53.0.

I'd probably go with the second option to avoid an unnecessary info callout.

Reply:
Done.